playground/mocks/data/idp/idx/enroll-profile-new.json

```diff
@@ -19,13 +19,13 @@
             "form": {
               "value": [
                 {
-                  "name": "lastName",
-                  "label": "Last name",
+                  "name": "firstName",
+                  "label": "First name",
```

Comment:
Was this required to change to make it work? My only concern is that this mock file is from v2, not sure if they expect specific order of these inputs.

Reply:
i changed the mock file back to its original state and got the test working by changing the expected focus element to lastName since it is being rendered first

Reply:
Awesome! Ty!